src/pages/documents/[id].tsx

```diff
@@ -285,76 +273,57 @@ const DocumentPage: InferGetServerSidePropsType<typeof getServerSideProps> = ({
-                            <SearchForm
-                              placeholder="Search the full text of the document"
-                              handleSearchInput={handleSearchInput}
-                              input={qsSearchString as string}
-                              size="default"
-                            />
-                          </div>
-                          <div className="relative z-10 flex justify-center">
-                            <button
-                              className="px-4 flex justify-center items-center text-textDark text-xl"
-                              onClick={() => setShowOptions(!showOptions)}
-                            >
-                              <MdOutlineTune />
-                            </button>
-                            <AnimatePresence initial={false}>
-                              {showOptions && (
-                                <motion.div
-                                  key="content"
-                                  initial="collapsed"
-                                  animate="open"
-                                  exit="collapsed"
-                                  variants={{
-                                    collapsed: { opacity: 0, transition: { duration: 0.1 } },
-                                    open: { opacity: 1, transition: { duration: 0.25 } },
-                                  }}
-                                >
-                                  <SearchSettings
-                                    queryParams={router.query}
-                                    handleSearchChange={handleSemanticSearchChange}
-                                    setShowOptions={setShowOptions}
-                                  />
-                                </motion.div>
-                              )}
-                            </AnimatePresence>
-                          </div>
+                      <div id="document-search" className="flex items-start gap-2 md:pl-4 pb-4 border-b border-gray-200">
+                        <div className="flex-1">
+                          {totalNoOfMatches > 0 && (
+                            <>
+                              <div className="mb-2 pt-2 text-sm" data-cy="document-matches-description">
+                                Displaying {renderPassageCount(totalNoOfMatches)} for "
+                                <span className="text-textDark font-medium">{`${qsSearchString}`}</span>"
+                                {!searchQuery.exact_match && ` and related phrases`}
+                                {totalNoOfMatches >= MAX_RESULTS && (
+                                  <span className="ml-1 inline-block">
+                                    <SearchLimitTooltip colour="grey" />
+                                  </span>
+                                )}
+                              </div>
+                              <p className="text-sm">Sorted by search relevance</p>
```

Comment:
❓question: what if this is sorted by date etc?

Reply:
There is no sorting available on the document page - if there was, this would need updating. (This text is unchanged in this PR, it's just code being shuffled).